### PR TITLE
Jordan/few gov fixes

### PR DESCRIPTION
**Pull request by jbibla**
**Description:**
- hide vote display during deposit period
- few style adjustments
- switch "preliminary proposal" to "proposal"

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
🚀 Deployed on https://5f661c7ca70c2da411c0b568--lunieio.netlify.app

File: api/lib/reducers/polkadotV0-reducers.js [hunk 1]
```diff
@@ -541,7 +541,7 @@ function democracyProposalReducer(
     proposalId: proposal.index,
     networkId: network.id,
     type: proposalTypeEnum.PARAMETER_CHANGE,
-    title: `Preliminary Proposal #${proposal.index}`,
+    title: `Democracy Proposal #${proposal.index}`,
```
Comment (faboweb) [inline api/lib/reducers/polkadotV0-reducers.js:544]: Why the change?

Comment (Bitcoinera) [reply api/lib/reducers/polkadotV0-reducers.js:544]: So the url and the title are consistent, as I understood it on Slack

File: app/src/components/governance/ProposalStatusBar.vue [hunk 2]
```diff
@@ -37,7 +37,10 @@
         bar-color="var(--highlight)"
       />
     </div>
-    <div class="bottom row">
+    <div
+      v-if="status.value !== governanceStatusEnum.DEPOSITING"
```
Comment (Bitcoinera) [inline app/src/components/governance/ProposalStatusBar.vue:41]:
```suggestion
      v-if="status.value === governanceStatusEnum.VOTING"
```

It only makes actually sense when it is on Voting Period

File: app/src/styles/app.css [hunk 3]
```diff
@@ -99,7 +99,7 @@ input.tm-field {
   padding: 7px;
   font-size: var(--sm);
   text-transform: uppercase;
-  top: 0;
+  top: 2px;
```
Comment (Bitcoinera) [inline app/src/styles/app.css:102]: And this change?

File: app/src/styles/app.css [hunk 4]
```diff
@@ -188,4 +188,4 @@ input.tm-field {
 
 .intercom-button {
   cursor: pointer;
-}
+}
```
Comment (Bitcoinera) [inline app/src/styles/app.css:191]: :eyes: can you please try linting?

File: api/lib/reducers/polkadotV0-reducers.js [hunk 5]
```diff
@@ -174,8 +174,8 @@ function delegationReducer(network, delegation, validator, active) {
     validator,
     amount: delegation.value
       ? BigNumber(delegation.value)
-          .times(network.coinLookup[0].chainToViewConversionFactor)
-          .toFixed(6)
+        .times(network.coinLookup[0].chainToViewConversionFactor)
+        .toFixed(6)
       : 0,
```
Comment (Bitcoinera) [inline api/lib/reducers/polkadotV0-reducers.js:179]: Why this API changes? Linting is not being consistent?

Comment (jbibla) [reply api/lib/reducers/polkadotV0-reducers.js:179]: yep just linting inconsistencies

File: app/src/styles/app.css (hunk 3 above)
Comment (jbibla) [reply app/src/styles/app.css:102]: the denom in TmField was not vertically centered

File: app/src/styles/app.css (hunk 4 above)
Comment (jbibla) [reply app/src/styles/app.css:191]: 😭

File: app/src/components/governance/ProposalStatusBar.vue (hunk 2 above)
Comment (jbibla) [reply app/src/components/governance/ProposalStatusBar.vue:41]: don't we want to show the vote for passed and rejected proposals?

Comment (Bitcoinera) [reply app/src/components/governance/ProposalStatusBar.vue:41]: As Fabo said, we cannot display those votes

Comment (jbibla) [reply app/src/components/governance/ProposalStatusBar.vue:41]: we can. we cannot show the percentages:
Screen Shot 2020-09-21 at 10 35 48 AM
.

Comment (Bitcoinera) [reply app/src/components/governance/ProposalStatusBar.vue:41]: Ah, ok. Cool

**Comment by Bitcoinera** [comment]
Is this one ready?

File: api/lib/reducers/polkadotV0-reducers.js (hunk 1 above)
Comment (faboweb) [reply api/lib/reducers/polkadotV0-reducers.js:544]: Preliminary is actually a good simplification for the user here. Can we solve this a different way?

**Review by mariopino — DISMISSED** [review]

File: api/lib/reducers/polkadotV0-reducers.js [hunk 6]
```diff
@@ -541,7 +541,7 @@ function democracyProposalReducer(
     proposalId: proposal.index,
     networkId: network.id,
     type: proposalTypeEnum.PARAMETER_CHANGE,
-    title: `Preliminary Proposal #${proposal.index}`,
+    title: `Proposal #${proposal.index}`,
```
Comment (Bitcoinera) [inline api/lib/reducers/polkadotV0-reducers.js:551]: Now refendums and democracies are called the same? Well, guess it kinds of makes sense, yes
